scalableminds/webknossos
# Create new export modal

### URL of deployed dev instance (used for testing):
https://createnewexportmodal.webknossos.xyz/

### Steps to test:
- open an annotation and select Download from the tracing actions
- the new modal should show with tabs for Download, Export and Python Client

### Issues:
- fixes #5672 

------
(Please delete unneeded items, merge only when none are left open)
